Content.Server/Speech/EntitySystems/OwOAccentSystem.cs

```diff
@@ -1,41 +1,175 @@
 using Content.Server.Speech.Components;
-using Robust.Shared.Random;
+using System.Text.RegularExpressions;
 
 namespace Content.Server.Speech.EntitySystems
 {
```

Comment:
You could use file scoped namespace here. So rather than:

```csharp
namespace Content.Server.Speech.EntitySystems
{
     public sealed class OwOAccentSystem
}
```

you write:

```csharp
namespace Content.Server.Speech.EntitySytems;

public sealed class OwOAccentSystem : EntitySystem
```

It's just one less indent level.

Reply:
I don't dislike this suggestion, but I don't think this is the standard in the codebase at the moment, so I'm opting not to do this to maintain consistency with the rest where I can.

Reply:
I did say it's a maybe. Most new code uses file scoped. And Rider offers this fix.

Reply:
On further review, it looks like this _is_ more common in the codebase, so I'll adopt this suggestion when I'm home